### PR TITLE
Add core weekly 45

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: news/_posts/core-weekly/2020-11-09-coreweekly-45-2020.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+### Email Alerts module
+* [#65](https://github.com/PrestaShop/ps_emailalerts/pull/65): Fix cast error when removing an Alert. Thank you [@hherreros93](https://github.com/hherreros93)
+* [#57](https://github.com/PrestaShop/ps_emailalerts/pull/57): Added new translator for email subjects. Thank you [@L3RAZ](https://github.com/L3RAZ)
+
+
+### Visits and Visitors statistics module
+* [#11](https://github.com/PrestaShop/statsvisits/pull/11): Release 2.0.3, by [@Progi1984](https://github.com/Progi1984)
+
+## Logo and description changes
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-11-09-coreweekly-45-2020.md:201]: this one is empty

Comment (PierreRambaud) [reply news/_posts/core-weekly/2020-11-09-coreweekly-45-2020.md:201]: No, it's the title with two `#` then you see all modules with three `#`.

**Review by PierreRambaud — APPROVED** [review]

File: news/_posts/core-weekly/2020-11-09-coreweekly-45-2020.md (hunk 1 above)
Comment (ttoine) [reply news/_posts/core-weekly/2020-11-09-coreweekly-45-2020.md:201]: My bad. I didn't understand that was a specific section for this weekly.

**Review by ttoine — APPROVED** [review]

**Comment by matks** [comment]
Thank you for review!

**Pull request merged**